From 7ce34966a7b374299d82758ebc91bc7934695e50 Mon Sep 17 00:00:00 2001
From: dmeier <dmeier@90b65887-3827-0410-9a23-83215b262276>
Date: Wed, 11 Dec 2019 14:16:01 +0000
Subject: [PATCH 0905/1000] MANTIS 56794 MDCI - Support Media Type
 "ID_DOCUMENT"

added data status DS_TYPES_ID_DOCUMENT
added compType IDDR
added dataType DT_ID_DOCUMENT


git-svn-id: svn://localhost/SelfServiceCommon/trunk@1509 90b65887-3827-0410-9a23-83215b262276
---
 SelfServiceCommon/Massai/idl/Cuss/codes.idl    | 7 +++++++
 SelfServiceCommon/Massai/idl/Massai/gmdev.idl  | 3 ++-
 SelfServiceCommon/Massai/idl/Massai/gmrdev.idl | 3 ++-
 3 files changed, 11 insertions(+), 2 deletions(-)

diff --git a/SelfServiceCommon/Massai/idl/Cuss/codes.idl b/SelfServiceCommon/Massai/idl/Cuss/codes.idl
index 8810abfe..bfd4fc08 100644
--- a/SelfServiceCommon/Massai/idl/Cuss/codes.idl
+++ b/SelfServiceCommon/Massai/idl/Cuss/codes.idl
@@ -98,6 +98,11 @@ Changed:   Added baggage handling status codes (Nicolas Bruns, 29.06.2004)
 @li Mantis 0055465: CUSS 1.5.1 Biometrics Interface
 - added data status DS_TYPES_BIOMETRIC
 
+@date 11.12.2019
+@li V1.5.1
+@li Mantis 0056794: MDCI - Support Media Type "ID_DOCUMENT"
+- added data status DS_TYPES_ID_DOCUMENT
+
 
 */
 
@@ -235,6 +240,8 @@ module datastatus
     const long DS_TYPES_CYPHER_CHALLENGE         = 40030; /**< random challenge for key generation */
     const long DS_TYPES_CYPHER_KEYS              = 40040; /**< encryption keys */
     const long DS_TYPES_CYPHER_TRACK             = 40050; /**< encrypted track data */
+	
+	const long DS_TYPES_ID_DOCUMENT              = 19500; /**< id document */
 };
 
 /**
diff --git a/SelfServiceCommon/Massai/idl/Massai/gmdev.idl b/SelfServiceCommon/Massai/idl/Massai/gmdev.idl
index c899e1ca..6f10a90f 100644
--- a/SelfServiceCommon/Massai/idl/Massai/gmdev.idl
+++ b/SelfServiceCommon/Massai/idl/Massai/gmdev.idl
@@ -57,7 +57,8 @@ module multidevices
     DLR,        /**< Driver license Reader */
     FACE,       /**< Face scanner device */
     EPPR,       /**< e-Passport */
-    SCR         /**< secure card reader */
+    SCR,        /**< secure card reader */
+    IDDR        /**< ID document reader */
   };
 
 };
diff --git a/SelfServiceCommon/Massai/idl/Massai/gmrdev.idl b/SelfServiceCommon/Massai/idl/Massai/gmrdev.idl
index b26a3357..235d0278 100644
--- a/SelfServiceCommon/Massai/idl/Massai/gmrdev.idl
+++ b/SelfServiceCommon/Massai/idl/Massai/gmrdev.idl
@@ -158,7 +158,8 @@ using DeviceComponents::indicationListener and DeviceComponents::Generic from md
     DT_CYPHER_HW_CERTIFICATE,  /* certifcate of the hardware */
     DT_CYPHER_CHALLENGE,       /* random challenge for key generation */
     DT_CYPHER_KEYS,            /* encryption keys */
-    DT_CYPHER_TRACK            /* encrypted track data */
+    DT_CYPHER_TRACK,           /* encrypted track data */
+    DT_ID_DOCUMENT             /* id document */
   };                      
 
   /**  The data status */
-- 
2.41.0.windows.1

